package.json

```diff
@@ -61,7 +61,7 @@
     "auto-start-ganache": "cross-env arcjs_network=ganache run-with-ganache --ganache-cmd ganache-cli \"npm run migrate-ganache && npm run start-ganache\""
   },
   "dependencies": {
-    "@daostack/arc.js": "0.0.0-alpha.58",
+    "@daostack/arc.js": "0.0.0-alpha.59",
```

Comment:
Why do you need the latest arc.js?

Reply:
To fix a bug in retrieving the contribution reward proposal description hash

Reply:
ok